### PR TITLE
Edit Docusaurus Documentation for Clarity

### PR DESCRIPTION
**Pull request by JerrettDavis**

### CODE REVIEW

File: docs/site/angular-auth-oidc-client/docs/documentation/configuration.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ Prior to using the library, you must configure it with the appropriate values fo
 You can pass the static config with the `config` property into the `forRoot()` method like this
 
 ```ts
-import { NgModule } from '@angular/core';
-import { AuthModule } from 'angular-auth-oidc-client';
+import { NgModule } from '@Angular/core';
```
Comment (FabianGosebrink) [inline docs/site/angular-auth-oidc-client/docs/documentation/configuration.md:15]: Imports are lowercase, as the folder in node_modules is

File: docs/site/angular-auth-oidc-client/docs/documentation/configuration.md [hunk 2]
```diff
@@ -150,207 +151,214 @@ Starts the OpenID session management for this client.
...
+Default = *3*. <br/>
+`silentRenewTimeoutInSeconds` and `tokenRefreshInSeconds` are the upper bounds for this value.
 
 ### `ngswBypass`
 
 - Type: boolean
 - Required: false
 
-Adds the `ngsw-bypass` param to all requests ([Angular Docu](https://angular.io/guide/service-worker-devops#bypassing-the-service-worker)).
+Adds the `ngsw-bypass` param to all requests ([Angular Documentation](https://Angular.io/guide/service-worker-devops#bypassing-the-service-worker)).
```
Comment (FabianGosebrink) [inline docs/site/angular-auth-oidc-client/docs/documentation/configuration.md:364]: `Angular` --> `angular`. Is not important as URLs are case insensitive but could make an SEO difference.

File: docs/site/angular-auth-oidc-client/docs/documentation/configuration.md (hunk 1 above)
Comment (JerrettDavis) [reply docs/site/angular-auth-oidc-client/docs/documentation/configuration.md:15]: Whoops, good catch. Had a bad replace-all there.

File: docs/site/angular-auth-oidc-client/docs/documentation/configuration.md (hunk 2 above)
Comment (JerrettDavis) [reply docs/site/angular-auth-oidc-client/docs/documentation/configuration.md:364]: Whoops, same replace-all issue from above.

**Pull request merged**